persistence-api/src/main/java/io/stargate/db/Persistence.java

```diff
@@ -34,10 +32,7 @@
  * higher level {@link DataStore} API which wraps an instance of this interface) to query the
  * underlying store, and thus the one interface that persistence extensions must implement.
  */
-public interface Persistence {
-
-  int SCHEMA_AGREEMENT_WAIT_RETRIES =
-      Integer.getInteger("stargate.persistence.schema.agreement.wait.retries", 900);
+public interface Persistence extends SchemaAgreementChecker {
```

Comment:
I guess you should now remove the declaration of the `boolean isInSchemaAgreement();` as it's now part of the `SchemaAgreementChecker` interface?

Reply:
done.